Lib/ipaddress.py

```diff
@@ -1146,11 +1141,11 @@ class _BaseV4:
     """
 
     __slots__ = ()
-    _version = 4
+    version = _version = 4
```

Comment:
I couldn't use classmethod, since that functionality was removed in 3.13: https://docs.python.org/3.13/library/functions.html#classmethod